action.yml

```diff
@@ -34,6 +35,14 @@ inputs:
     description: Show hash values of files to be uploaded
     required: false
     default: 'false'
+  dry_run:
+    description: Do everything up to the final package upload
+    required: false
+    default: 'false'
```

Comment:
NB: I'll remove this before merge; just added it here to make my own testing a little easier 🙂